README.md

```diff
@@ -52,6 +52,9 @@ On the new component panel, copy and paste the following attribute template into
   }
 }
 ```
+
+Additionally, make sure to add your configured data manager service to the `depends_on` array of your `video-store` component.
```

Comment:
TBH we should make data manager an implicit dep using `Validate`

Reply:
Hm yeah that would be nice to reduce setup burden -- I thought we could only do this using builtins?

can we just do something similar to this https://github.com/viamrobotics/rdk/blob/main/components/input/gpio/gpio.go#L49-L59

Reply:
I think I was able to do it in a Python module ?

Reply:
anyways out of scope here